nixos/modules/security/pam.nix

```diff
@@ -207,6 +207,14 @@ let
         '';
       };
 
+      enableFscrypt = mkOption {
+        default = config.security.pam.enableFscrypt;
```

Comment:
Update3: It seems to me like the best approach to implement this would be `default = false` and set ` security.pam.services.$name.enableFscrypt = config.security.pam.enableFscrypt` (the actual implementation would be a bit different) for the relevant services. But I'm not sure how realistic that approach currently is for NixOS.